### PR TITLE
Migrate JPA Packaging Bucket to OL

### PR DESCRIPTION
**Pull request by jgrassel**
Migrate the JPA Packaging bucket to Open Liberty

### CODE REVIEW

**Comment by jgrassel** [comment]
#build

File: dev/com.ibm.ws.jpa.tests.spec10.packaging_fat.common/fat/src/com/ibm/ws/jpa/tests/spec10/packaging/tests/Packaging_Web.java [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+import componenttest.topology.database.container.DatabaseContainerType;
+import componenttest.topology.database.container.DatabaseContainerUtil;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.PrivHelper;
+
+@RunWith(FATRunner.class)
+@Mode(TestMode.LITE)
+public class Packaging_Web extends JPAFATServletClient {
+    @Rule
+    public org.junit.rules.TestRule skipDBRule = new org.junit.rules.Verifier() {
```
Comment (dazey3) [inline dev/com.ibm.ws.jpa.tests.spec10.packaging_fat.common/fat/src/com/ibm/ws/jpa/tests/spec10/packaging/tests/Packaging_Web.java:54]: Maybe migrate this to `com.ibm.ws.jpa.tests.spec10.packaging_fat.common/fat/src/com/ibm/ws/jpa/tests/spec10/packaging/tests/SkipDatabaseRule.java`? I don't see `SkipDatabaseRule` being used anywhere.

File: dev/com.ibm.ws.jpa.tests.spec10.packaging_fat.common/test-applications/packaging/Packaging10EJB.ear/JPAAnoOverrideEJB.jar/META-INF/persistence.xml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<persistence xmlns="http://java.sun.com/xml/ns/persistence"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_1_0.xsd"
+    version="1.0">
+
+    <persistence-unit name="Cleanup" transaction-type="RESOURCE_LOCAL">
```
Comment (dazey3) [inline dev/com.ibm.ws.jpa.tests.spec10.packaging_fat.common/test-applications/packaging/Packaging10EJB.ear/JPAAnoOverrideEJB.jar/META-INF/persistence.xml]: Do these "Cleanup" persistence units get used anywhere in the application? I don't see any injections for them.

**Review by dazey3 — COMMENTED** [review]
suggestions

File: dev/com.ibm.ws.jpa.tests.spec10.packaging_fat.common/test-applications/packaging/Packaging10EJB.ear/JPAAnoOverrideEJB.jar/META-INF/persistence.xml (hunk 2 above)
Comment (jgrassel) [reply dev/com.ibm.ws.jpa.tests.spec10.packaging_fat.common/test-applications/packaging/Packaging10EJB.ear/JPAAnoOverrideEJB.jar/META-INF/persistence.xml]: nah they can go.

File: dev/com.ibm.ws.jpa.tests.spec10.packaging_fat.common/fat/src/com/ibm/ws/jpa/tests/spec10/packaging/tests/Packaging_Web.java (hunk 1 above)
Comment (jgrassel) [reply dev/com.ibm.ws.jpa.tests.spec10.packaging_fat.common/fat/src/com/ibm/ws/jpa/tests/spec10/packaging/tests/Packaging_Web.java:54]: Will just use the inline and get rid of the copy of SkipDatabaseRule.

**Comment by jgrassel** [comment]
#runlibbybot
#build

**Comment by jgrassel** [comment]
Unrelated spnego test failed, otherwise good

**Review by dazey3 — APPROVED** [review]
lgtm

**Comment by jgrassel** [comment]
#runlibbybot

**Comment by jgrassel** [comment]
#libbybot

**Pull request merged**